### PR TITLE
translate <delpoy-ha>/beehive from English to Chinese with help of chatgpt

### PR DESCRIPTION
**Pull request by magichan**
* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Which issue(s) this PR fixes**:

Fixes #
https://github.com/kubeedge/website/issues/619

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Bug fix

* **What is the current behavior?** (You can also link to an open issue here)

Display English documents in Chinese mode

* **What is the new behavior (if this is a feature change)?**

translate <delpoy-ha> from English to Chinese

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

No

* **Other information**:

### CODE REVIEW

**Comment by magichan** [comment]
Some Question:

1. Is Ingress-based HA implemented?

**Comment by magichan** [comment]
@kevin-wangzefeng  

Is it convenient to review the Chinese translations of the existing three documents?

The other documents are expected to be completed before the next community meeting begins, but I do not expect to have a huge MR merge. I hope to split several relatively small MR merges.

File: docs/architecture/cloud/edge_controller.md [hunk 1]
```diff
@@ -5,7 +5,10 @@ sidebar_position: 2
 
 
 ## Edge Controller Overview
- EdgeController is the bridge between Kubernetes Api-Server and edgecore
+
+**Note**: This document is generally based on version 1.5.0. Although the main functionality and core processes remain unchanged in the new version, some technical details have changed.
```
[the remark this answers is not in the document]
Comment (magichan) [reply docs/architecture/cloud/edge_controller.md:9]: for it is repeated with https://github.com/kubeedge/website/pull/618 , remove the content.

**Comment by magichan** [comment]
Based on the discussion at the community meeting,  edge-controller was removed because it was a duplicate of https://github.com/kubeedge/website/pull/618

**Review by Shelley-BaoYue — COMMENTED** [review]
Please solve the DCO.

**Comment by magichan** [comment]
> Please solve the DCO.

@Shelley-BaoYue  Thx, and Done.

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Comment by kevin-wangzefeng** [comment]
/approve
Thanks

**Pull request merged**